source/grid/grid_in.cc

```diff
@@ -1780,49 +1780,35 @@ GridIn<dim, spacedim>::read_msh(std::istream &in)
                   >> cell_type;  // ELM-TYPE
               }
 
-            switch (gmsh_file_format)
+            if (gmsh_file_format < 20)
               {
-                case 10:
```

Comment:
In the original code, under what condition was it possible that `gmsh_file_format` was less than 10?

Reply:
Because looking at lines 1422 - 1428 suggests that this is not possible, or am I blind? :-D

Reply:
The issue was that `gmsh_file_format` was 22. Previously, "2.2" was reduced to "2" but now we multiply by 10 and store that as `gmsh_file_format`.

Reply:
Ah, what a fluke :-D